docs/modules/ROOT/pages/advanced-how-tos/building-olm.adoc

```diff
@@ -0,0 +1,27 @@
+= Building an Operator Lifecycle Manager (OLM) in {ProductName}
+
+If you are using an link:https://olm.operatorframework.io/docs/[Operator Lifecycle Manager (OLM)] for your application, you can build the OLM in {ProductName}. The first procedure in this document explains how to use {ProductName} to build an OLM in the most basic sense--building its operator and bundle. The second procedure is optional but recommended. It explains how you can automatically update the image references to the operator and operands that should be present in your bundle.
```

Comment:
I would replace `an` for `the` before the OLM link. Also, you don't build "the OLM", you build an operator that is managed by the OLM

Reply:
In this case, `an` would be appropriate, since the instructions apply to any Operator you want to build in Konflux. However, you were clearly right about the OLM vs Operator distinction. I applied your other suggested changes. Please let me know if the language looks better now.

(The main remaining challenge was to differentiate between "Operator" meaning the whole package, and "operator" meaning the controller. So I called the single component that gets made into an image the "controller," and referred to the whole application as an "Operator," which I think is consistent with the Operator Framework: https://operatorframework.io/)

Reply:
I still don't see it here! Because you are not saying "an operator", but "an Operator Lifecycle Manager". Maybe you could say something like: "If you are using the Operator Lifecycle Manager (OLM) for your operator, you can build it and its bundle in Konflux". Or something along those lines. Because here you are saying that you are using "an" OLM, and there are not several OLMs out there. And after that you say "build the OLM", and that means building the framework itself. At least that's what I understand

Reply:
Forget my last comment. I saw the updated paragraph!

Reply:
@Chr1st1anSears I made some comments. I like the changes but using "controller" as the word for the operator image is not correct either. Because an operator could have several controllers. I commented two of those but there are more in the docs. 

Operator: is the software that deploys and manages the lifecycle of complex applications in Kubernetes clusters. It could include several controllers when there are several operands to manage.

Bundle: image that includes the operator and all the manifests needed to deploy it